src/DocumentDB.ChangeFeedProcessor/PartitionManagement/RemainingWorkEstimator.cs

```diff
@@ -83,6 +85,25 @@ public async Task<IReadOnlyList<RemainingPartitionWork>> GetEstimatedRemainingWo
             return results.SelectMany(r => r).ToList().AsReadOnly();
         }
 
+        internal static string ExtractLsnFromSessionToken(string sessionToken)
```

Comment:
Let's provide short example of session token and explain where PKRangeId and GlobalLSN are. Don't need to provide internal details.